.circleci/config.yml

```diff
@@ -385,6 +385,116 @@ jobs:
       - store_artifacts:
           path: packages/python/plotly/dist
 
+
+  build-doc:
+    resource_class: xlarge
+    docker:
+      # specify the version you desire here
+      # use `-browsers` prefix for selenium tests, e.g. `3.6.1-browsers`
+      - image: circleci/python:3.6.8-node
+
+
+        working_directory: ~/project
+
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - "a1:22:8b:da:c9:9f:88:db:be:cb:1d:c5:12:de:70:4f"
```

Comment:
I was not sure how this line is used but I copied it none the less, to be discussed.

Reply:
In order to be able to push to the `plotly.py-docs` repo, yu'll need to manually build a keypair on your machine and add the public key to Github as a read/write deploy key, and you'll need to give the private key to Circle. The fingerprint for that key appears here for Circle to be able to use it.

Unfortunately you'll have to do this whole dance because I'm pretty sure I deleted the keypair from my machine once I was done setting this up ;)

@jdamiba just set this up in the `plotly.r-docs` repo so he can help you!

Reply:
OK, thanks! Yeah, maybe it will be faster if @jdamiba can give me a hand :-).

Reply:
OK, so I generated the new pair of keys (thank you @jdamiba!) and did the part on circle CI (fingerprint updated on the last commit). Now to add the public part I think I need to be made admin of this repo since I cannot access the settings. Otherwise I can just send you @nicolaskruchten the key.

Reply:
ok done!